### ISSUE TITLE
Parsing fails for fortran doubles written with a + prefix

### ISSUE THREAD

**Issue by oponkork** [marshallward/f90nml#113]
Consider the following test namelist file
```fortran
&test_nml
alpha =  1.d+0
beta  = -1.d+0
gamma = +1.d+0
/
```
Reading it from `cli` with `f90nml` as `f90nml test.nml` gives the following output
```fortran
&test_nml
    alpha = 1.0
    beta = -1.0
    gamma = 1, '.d', 0
/
```
It reads `alpha` and `beta` properly, but fails to account for the `+` sign for `gamma`. If you write an integer with a `+` sign, for example `+100`, the output is `100` as expected.

f90nml 1.1.2

**Comment by marshallward** [comment]
This appears to have been fixed in the master branch since 1.1.2 was released.  Can you verify this for me?

Regardless, I will do a version update sometime in the next day or two.

**Comment by oponkork** [comment]
Works correctly with master. I thought `V 1.1.2` is in sync with master. My bad, I will close the issue.
